### PR TITLE
Api gateway -> User-mgmt communication

### PR DESCRIPTION
**Pull request by pjazdzewski1990**
For now builds a DB with user subscriptions, but doesn't use them in sending (this PR is big already)

### CODE REVIEW

File: user-mgmt/build.sbt [hunk 1]
```diff
@@ -9,5 +9,11 @@ lazy val root = (project in file(".")).
     )),
     name := "newspaper-user-mgmt",
     libraryDependencies ++= (akkaDependencies :+ schemaDependency),
+    libraryDependencies ++= dbDependencies,
     libraryDependencies += scalaTest % Test
   )
+
+flywayUrl := "jdbc:postgresql://192.168.99.100:5432/user_mgmt_db" //TODO: how to provide host?
```
Comment (rsekulski-scalac) [inline user-mgmt/build.sbt]: How about moving migration from sbt task to application start-up?

File: user-mgmt/project/Dependencies.scala [hunk 2]
```diff
@@ -9,6 +9,15 @@ object Dependencies {
     "com.typesafe.akka" %% "akka-stream-kafka" % "0.14",
     "com.typesafe" % "config" % "1.3.1"
   )
+
+  lazy val slickVersion = "3.2.0"
+  lazy val dbDependencies = Seq(
+    "org.postgresql" % "postgresql" % "42.1.1",
+    "com.typesafe.slick" %% "slick" % slickVersion,
+    "org.slf4j" % "slf4j-nop" % "1.6.4",
+    "com.typesafe.slick" %% "slick-hikaricp" % slickVersion
```
Comment (rsekulski-scalac) [inline user-mgmt/project/Dependencies.scala]: Do you use ConnectionPool? I've seen it disabled in config.

File: newspapaer-api-gateway/build.sbt [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+lazy val root = (project in file(".")).enablePlugins(PlayScala)
+
+scalaVersion := "2.11.11"
+
+lazy val akkaVersion = "2.5.2"
+
+libraryDependencies += guice
+libraryDependencies += "com.typesafe.akka" %% "akka-actor" % akkaVersion
+libraryDependencies += "com.typesafe.akka" %% "akka-testkit" % akkaVersion
```
Comment (rsekulski-scalac) [inline newspapaer-api-gateway/build.sbt]: Not needed right now? Can be marked as a test dependency

File: user-mgmt/build.sbt (hunk 1 above)
Comment (pjazdzewski1990) [inline user-mgmt/build.sbt]: Will do

File: newspapaer-api-gateway/build.sbt (hunk 3 above)
Comment (pjazdzewski1990) [inline newspapaer-api-gateway/build.sbt]: 👍

File: user-mgmt/project/Dependencies.scala (hunk 2 above)
Comment (pjazdzewski1990) [inline user-mgmt/project/Dependencies.scala]: 👍

**Pull request merged**